af/analysis/ooni_db_backup.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+OONI DB backup tool
+Configured with /etc/ooni/db-backup.conf - Runs as a system daemon
```

Comment:
Is it possible to have an example `db-backup.conf` file checked in?

Reply:
Added.